src/features/courses/course-route.ts

```diff
@@ -235,7 +235,7 @@ router.post('/question/:id',
         proxyReqPathResolver: (req) => {
             return `/rendered?${qs.stringify({
                 format: 'json',
-                template: 'simple',
+                template: 'single',
```

Comment:
This is good but I think we also need this on get question (in the file `renderer-helper.ts`